test/Microsoft.ML.TimeSeries.Tests/TimeSeriesEstimatorTests.cs

```diff
@@ -41,7 +41,7 @@ public TimeSeriesEstimatorTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))] // netcore3.0 output differs from Baseline
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCoreAnd64BitProcess))] // 32bit and netcore3.0 output differs from Baseline
```

Comment:
This change seems unrelated. Why is it necessary?

Reply:
Our new algorithm leads to slightly different numbers due to different number of multiplications. (in case of floating points). After the change the x86 baseline numbers are almost same as netcoreapp but different from the original x64